src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1335,6 +1341,9 @@ void start(final BundleContext bundleContext)
                     bundleContext,
                     ConfigurationService.class);
 
+        videobridgeExpireThread.start(bundleContext);
+        videobridgeExpireThread.start();
```

Comment:
Not really mandatory, but what do you guys think if we called `start` from within `start(bundleContext)`?